src/EditorFeatures/Core/Implementation/Suggestions/SuggestedAction.cs

```diff
@@ -168,14 +210,19 @@ void IDisposable.Dispose()
             // do nothing
         }
 
-        public virtual IEnumerable<SuggestedActionSet> ActionSets
+        public virtual bool HasActionSets
         {
             get
             {
-                return null;
+                return false;
             }
         }
 
+        public virtual Task<IEnumerable<SuggestedActionSet>> GetActionSetsAsync(CancellationToken cancellationToken)
+        {
+            return SpecializedTasks.Default<IEnumerable<SuggestedActionSet>>();
+        }
```

Comment:
It looks like the behavior matches the previous `ActionSets` property. That said:

:exclamation: I would prefer this not return null in its current form. I would expect clear documentation listing null as a potential return value (along with its meaning), or a name like `TryGetActionSetsAsync`, which suggests the result might not be available where `null` would be an expected potential value.